### PR TITLE
Remove calibrations and retries from signal analyzers 

### PR DESCRIPTION
**Pull request by dboulware**
This removes the calibration handling from the signal analyzer so the scaling can occur within the Sensor class. In addition, retry behavior is removed from signal analyzer implementation so it can be handled at the Sensor level. This also updates the readme to address the signal analyzer related settings.

### CODE REVIEW

File: tests/test_tekrsa_sigan.py
Comment (jhazentia) [inline tests/test_tekrsa_sigan.py:1]: Comment on Line 21 should be updated as there should not be any test cal file here

File: src/scos_tekrsa/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "5.0.2"
```
Comment (jhazentia) [inline src/scos_tekrsa/__init__.py:1]: Should this be a major version change due to breaking changes in TekRSASigan constructor and acquire_time_domain_samples method signatures?

**Review by jhazentia — CHANGES_REQUESTED** [review]
Looks good, just a couple of minor comments

File: src/scos_tekrsa/__init__.py (hunk 1 above)
Comment (aromanielloNTIA) [reply src/scos_tekrsa/__init__.py:1]: Yes, bumped to 6.0.0 in cdd9f1871ee1aa0643dc98ad0c6a4cf19f5098a9

File: tests/test_tekrsa_sigan.py
Comment (aromanielloNTIA) [reply tests/test_tekrsa_sigan.py:1]: Updated in 32b27971d08bdcb32aff53ff0aade89c5562bd1e

**Review by jhazentia — APPROVED** [review]

**Pull request merged**